pkgs/misc/tmux-plugins/default.nix

```diff
@@ -534,6 +534,24 @@ in rec {
     };
   };
 
+  rose-pine = mkTmuxPlugin {
+    pluginName = "rose-pine";
+    version = "unstable-2024-01-08";
+    rtpFilePath = "rose-pine.tmux";
```

Comment:
Is it necessary?

Reply:
Yes, otherwise we get `rose_pine.tmux` which doesn't exist.